docs/src/main/asciidoc/datasource.adoc

```diff
@@ -382,6 +403,10 @@ quarkus.datasource.inventory.jdbc.max-size=12
 Notice there is an extra bit in the key.
 The syntax is as follows: `quarkus.datasource.[optional name.][datasource property]`.
 
+NOTE: Even when only one database extension is installed named databases need to specify at least one build time
+property so that Quarkus knows they exist. Generally this will be the `db-kind` property, although you can also
```

Comment:
Yeah, typically, that's one of the reasons why I didn't like having `db-kind` being optional. I'm still not convinced it's a good idea. I suppose we will see...

Reply:
The reason why I think it is ok is that the single DB + single driver is the 99% use case, and we can effectively make this configuration free.